pkg/controller.v1/mpi/mpijob_controller.go

```diff
@@ -1152,6 +1152,26 @@ func (jc *MPIJobReconciler) newLauncher(mpiJob *kubeflowv1.MPIJob, kubectlDelive
 			})
 	}
 
+	// Add default Intel MPI bootstrap variables if not provided by the user.
+	if bootstrap, exec := hasIntelMPIBootstrapValues(container.Env); !bootstrap || !exec {
```

Comment:
```suggestion
	if bootstrap, exec := hasIntelMPIBootstrapValues(container.Env); {
```

Redundantly checking wouldn't need.

Reply:
Thanks, I removed the if clause completely.